griptape/drivers/prompt/claude_prompt_driver.py

```diff
@@ -0,0 +1,33 @@
+from typing import Optional
+import anthropic
+from attr import define, field, Factory
+from griptape.artifacts import TextArtifact
+from griptape.drivers import BasePromptDriver
+from griptape.tokenizers import TiktokenTokenizer
+
+
+@define
+class ClaudePromptDriver(BasePromptDriver):
```

Comment:
Let's call it `AnthropicPromptDriver`, since there will likely be other models from Anthropic in the future.